python/pricing_solver_example.py

```diff
@@ -0,0 +1,32 @@
+import get_table_list
+import load_data_tables
+import load_decrement_tables as decrement
+import pricing_solver
+import read_data_tables
+import logging
+import setup
+
+if __name__ == '__main__':
+    # Change this to appropriate level for your run
+    setup.setup_logging(logging.INFO)
+
+    # Get List of Data Tables
+    get_table_list.get_table_list()

```

Comment:
Consider pulling these first 4 calls into another example script that just shows various examples.  They do not seem to be directly related to the pricing solver.  Could even just make each of these other files their own examples.  

We might also consider renaming the examples or reorganizing (maybe in a folder?) so that they are all together.  I was thinking something like (if we go the naming route):

- example_get_table_list
- example_pricing_solver
- etc